src/ngx-smart-modal/src/components/ngx-smart-modal.component.ts

```diff
@@ -49,6 +59,7 @@ export class NgxSmartModalComponent implements OnInit, OnDestroy {
   @Input() public hideDelay: number = 500;
   @Input() public autostart: boolean = false;
   @Input() public target: string = '';
+  @Input() public closeIconColor: string = 'inherit';
```

Comment:
I don't think this input is necessary. If the developer wants to change the color, he can do so by using CSS. This is not a functional need of the modal.